jwtAuth.go

```diff
@@ -199,3 +221,10 @@ func GetClaims(request *http.Request) *TokenClaims {
 	}
 	return nil
 }
+
+/*
+IsVerifyToken is true when the AUTHENTICATION_VERIFY_TOKEN_SIGNATURE is properly set
+*/
+func IsVerifyToken() bool {
+	return os.Getenv("AUTHENTICATION_VERIFY_TOKEN_SIGNATURE") == "true"
```

Comment:
It should return `true` by default  because in the case you forget to set that env var, then your microservice's security level is lower than the maximum. This should not be the case. If a dev is clumsy, your microservice should be as well protected as possible.
Hence:
```suggestion
	return  ! os.Getenv("AUTHENTICATION_VERIFY_TOKEN_SIGNATURE") == "false"
```
You want to make sure that the devs is actually requesting that there should be no signature verification, not the other way around.